### DISCUSSION TITLE
Does this work with docker containers?

### DISCUSSION THREAD

**Discussion by jbxonline** [spatie/laravel-ray#56] [Q&A] [answered]
Looks like an amazing tool, but I'm not getting any output at all.

I'm running my Laravel 8 project inside a Docker Container, so I'm guessing that this is causing me an issue somehow?
Any advice would be appreciated, thanks :)
Jon

**Comment by freekmurze** [comment, marked as answer]
Have you tried using host `host.docker.internal`?

There's also an extra hint at the bottom of this page: https://spatie.be/docs/ray/v1/getting-started/configuring-ray

**Reply by jbxonline** [reply]
That got things working for me, thanks @freekmurze 
Definitely a case of RTFM 👍🏻

**Comment by victoralbino** [comment]
I use laradock for Docker, and even with the documentation settings I was not successful in using the tool
